src/bzflag/ShockWaveStrategy.cxx

```diff
@@ -32,13 +32,8 @@ ShockWaveStrategy::ShockWaveStrategy(const FiringInfo& f) :
 
     // make scene node
     const float* pos = getPosition();
-    if (RENDERER.useQuality() >= 2)
-    {
-        shockNode = new SphereLodSceneNode(pos, radius);
-        shockNode->setShockWave(true);
-    }
-    else
-        shockNode = new SphereBspSceneNode(pos, radius);
+    shockNode = new SphereSceneNode(pos, radius);
```

Comment:
this initialization could be moved to the initializer list, where it belongs:

```c++
    shockNode( new SphereSceneNode(getPosition(), radius) ),
```

Reply:
Done